package.json

```diff
@@ -9,8 +9,10 @@
         "url": "git@github.com:box/box-content-preview.git"
     },
     "devDependencies": {
-        "@babel/core": "^7.5.5",
+        "@babel/cli": "^7.18.6",
+        "@babel/core": "^7.18.6",
         "@babel/plugin-proposal-class-properties": "^7.5.5",
```

Comment:
Can we update the remaining plugins and presets, as well?

Reply:
Yup! Updated! :)